apps/hash-graph/README.md

```diff
@@ -31,7 +31,7 @@ cargo run -- server
 ### Logging configuration
 
 Some of the libraries used are very talkative in `trace` logging configurations, especially `mio`, `hyper`, and `tokio_util`.
-If you're interested in just increasing the logs for the Graph, we recommend specifically targeting the crates with `RUST_LOG=graph=trace,hash_graph=trace`.
+If you're interested in just increasing the logs for the Graph, we recommend specifically targeting the crates with `RUST_LOG=graph=trace,hash-graph=trace`.
```

Comment:
We have `RUST_LOG` values using `hash_graph` in:
- [apps/hash-external-services/docker-compose.yml#L93](https://github.com/hashintel/hash/blob/d5b9071524c3094059d48466fd32a59716b08a0d/apps/hash-external-services/docker-compose.yml#L93) (and line 145)
- [infra/terraform/hash/prod-usea1.tfvars#L33](https://github.com/hashintel/hash/blob/d5b9071524c3094059d48466fd32a59716b08a0d/infra/terraform/hash/prod-usea1.tfvars#L33)